Lib/urllib/parse.py

```diff
@@ -778,6 +781,15 @@ def parse_qs(qs, keep_blank_values=False, strict_parsing=False,
             parsed_result[name] = [value]
     return parsed_result
 
+def _is_valid_query(to_check: str) -> bool:
```

Comment:
```suggestion
def _is_valid_rfc3986_query(chars):
```

Note: we don't do type annotations in the standard library.